tests/integration/cli/test_init.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
+
+from tests.integration.cli.utils import run_once
+
+"""
+The purpose of this unit test is to test the funcationality of `ape init`
+
+It should test the creation of directories, config.yaml, and .gitginore:
+contracts
+test
+scripts
+ape-config.yaml
+.gitignore
+"""
+
+
+def test_github_clone():
+    pass
+
+
+@run_once
+def test_init_success(ape_cli, runner):
+    # Successfull creation of project
+    # ape init command
+
+    project_folder = Path.cwd()
```

Comment:
```suggestion
    project_folder = Path.cwd()
    os.system('rm -rf ./* && rm .gitignore')
```

Reply:
if you don't rm these files then there is some remaining from whatever project we are using already.